src/test/java/org/apache/ibatis/submitted/annotion_many_one_add_resultmapid/UserDao.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2019 ucsmy.com, All rights reserved.
+ */
```

Comment:
Same here.
Please remove the header if that is OK with your organization.

Reply:
ok,I will delete it